CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 21.2
+Enhance your WooCommerce experience with this release! Enjoy GTIN support for products, improved Tap To Pay testing, and refined order editing. Merchants now have control over campaign call-to-action text in Blaze. Plus, improved receipt communication, a refined dashboard view, and WordPress.com account setup in Jetpack.
```

Comment:
```suggestion
It's two weeks since the last one, so we've got lots for you in this release. The app now supports GTIN global identifiers for products, and you can now update the call-to-action text in your Blaze campaigns. It's quicker to get your store set up for in-person payments, easier to see when a receipt has been sent for a payment, and we've fixed some bugs when trying out Tap to Pay on iPhone. Lastly, we've made some fixes to the stats in the dashboard.
```

Reply:
Thanks so much for the suggested version, definitely reads better and more friendly 💟  The length is over the 350 characters, so I asked ChatGPT to shorten it like:

```
In just two weeks, we've packed this release with great updates. Enjoy GTIN support for products and customizable call-to-action text in Blaze campaigns. Store setup for in-person payments is faster, receipt confirmations clearer, and Tap to Pay on iPhone is smoother. Plus, we've improved dashboard statistics and squashed some bugs.
```

WDYT?

The 350 character recommendation is from the release task details:

> (Note: we try to keep the English copy under around 350 characters despite the limit in App Store being 4000, to keep it around the same length as its Android counterpart, which has a way smaller limit. Also remember that translations in some locales can end up lengthier than the English copy they are translating)

Reply:
Sorry, I didn't realise that was the cap, I'd remembered 500.

I tweaked it back a little:

> In just two weeks, we've jam-packed this release. There's GTIN global product identifier support, and you can edit the call-to-action in Blaze campaigns. 
> 
> Store setup for in-person payments is faster, receipt send confirmations clearer, and testing Tap to Pay on iPhone is smoother. Plus, we've improved dashboard statistics and squashed some bugs.

350 exactly

Reply:
Thanks a lot @joshheald! I made a minor edit from `receipt send confirmations` to `receipt sent confirmations`, will set the PR to auto-merge now.